airbyte-integrations/connectors/source-slack/source_slack/manifest.yaml

```diff
@@ -253,11 +253,75 @@ definitions:
               - channel_id
             value: "{{ stream_partition.get('channel') }}"
 
+  threads_stream:
+    $ref: "#/definitions/stream_base"
+    $parameters:
+      name: threads
+      path: conversations.replies
+      extractor_field_path: messages
+    primary_key:
+      - channel_id
+      - ts
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          channel: "{{ stream_slice['parent_slice']['channel'] }}"
+        error_handler:
+          type: DefaultErrorHandler
+          backoff_strategies:
+            - type: "WaitTimeFromHeader"
+              header: "retry-after"
+            - type: "WaitTimeFromHeader"
+              header: "Retry-After"
```

Comment:
This is probably fine, but I do find it interesting that in the other low-code streams we check lowercased `retry-after`, but in the python `SlackBackoffStrategy` we use `Retry-After`. The Slack API docs only reference `Retry-After`. 

So I'm quite curious if we've actually seen the lower-case retry header during a sync

Reply:
I didn't find in the cdk where it could be converted to lower case, asked Aldo about this as the author of the change.

Reply:
Aldo responded that there was a header in the actual response from the api